.github/workflows/build.yml

```diff
@@ -149,7 +149,7 @@ jobs:
     strategy:
       matrix:
         distro:
-          - jammy
+          - focal
```

Comment:
Focal Fossa is a _downgrade_ from Jammy Jellyfish. Is that what you really want?

Here's the recent list of Ubuntu branding names for reference:

- 24.04 Noble Numbat
- 22.04 Jammy Jellyfish
- 20.04 Focal Fossa
- 18.04 Bionic Beaver

Reply:
yes, because we flipped the default, but still support the old version 😉

Reply:
Ah okay! :smile: You can resolve this review comment then. :wink: